.gitignore

```diff
@@ -5,3 +5,4 @@ _site/**
 *.swo
 .rake_tasks
 .DS_Store
+Gemfile.lock
```

Comment:
Better to just commit the changes to gemfile.lock for now (or remove Gemfile.lock from the repo, but that runs the risk of us having multiple versions of the software that we use to build the site)